powerspikegg/computation_models/match/freeze.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+stats=(kills deaths assists minions_killed neutral_minions_killed total_damages total_heal wards_placed tower_kills) 
+
+models="/tmp/powerspikegg/models"
+
+for stat in ${stats[*]}; do
```

Comment:
use `"${stats[@]}"` to iterate over it. Without quotes, bash may interpret it as an empty array, leading in the statement `for stat in ; do` which may cause troubles...